rpcs3/Emu/Cell/PPUThread.h

```diff
@@ -125,7 +125,7 @@ class ppu_thread : public cpu_thread
 	virtual std::string dump_misc() const override;
 	virtual void cpu_task() override final;
 	virtual void cpu_sleep() override;
-	virtual ~ppu_thread() override;
+	virtual ~ppu_thread() override {};
```

Comment:
It should be more efficient to keep empty destructor in cpp file, to prevent compiling it without necessity.